cosmoz-data-nav.js

```diff
@@ -630,42 +658,36 @@
 			Polymer.IronResizableBehavior.notifyResize.call(this);
 		},
 
+		resizerShouldNotify() {
+			return false; //handle resize manually
+		},
+
 		/**
 		 * Notifies a descendant resizable of the element.
 		 *
 		 * @param  {HTMLElement} element The element to search within for a resizable
-		 * @return {void}
+		 * @return {Boolean|void} True if descendant has been notified.
```

Comment:
This should rather just be a Boolean return, we should return false instead of undefined if it's meant to be a signal back to the caller.

Reply:
@nomego Will change that.